src/pages/docs/go-w3up.mdx

```diff
@@ -0,0 +1,181 @@
+import { Callout } from 'nextra/components'
+import { Steps } from 'nextra/components'
+
+# `go-w3up`
+
+<Callout type="warning">
+  The Go client is under heavily development and is not as fully featured as the [JS client](/docs/w3up-client).
+</Callout>
+
+You can easily integrate web3.storage into your Go apps using `go-w3up`, our Go client for the w3up platform.
+
+In this guide, we'll walk through the following steps:
+
+1. [Installing the client library](#install)
+1. [Generating a DID](#generate-a-did)
+1. [Obtaining proofs](#obtain-proofs)
+1. [Loading your private key and proofs](#load-priviate-key-and-proofs)
+1. [Uploading a CAR file](#upload-a-car)
+
+## Install
```

Comment:
probably we should mention to install npx/npm

Reply:
It does say "Install Node.js" where it is needed below. I wanted to create a go equivalent so you'd not need to so would prefer to not mention here!